### PR TITLE
Validate that service uploads zipped PDFs in E2E test

### PR DESCRIPTION
**Pull request by banderous**
### Change description ###
Update the end to end test to validate that the files uploaded to xerox are zipped PDFs, as Xerox requires.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/sendletter/e2e/EndToEndTest.java [hunk 1]
```diff
@@ -84,10 +87,21 @@ private boolean letterHasBeenPosted() {
...
 
+    // Validate that the file is a zipped PDF,
+    // returning the Letter's ID from the filename.
+    private UUID validateGettingLetterId(File file) {
+        try {
+            PdfHelper.validateZippedPdf(Files.readAllBytes(file.toPath()));
+            return FileNameHelper.extractId(file.getName());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/sendletter/e2e/EndToEndTest.java]: or maybe just change test singature to `throws Exception`?

File: src/test/java/uk/gov/hmcts/reform/sendletter/PdfHelper.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import org.apache.pdfbox.preflight.parser.PreflightParser;
+import org.apache.pdfbox.preflight.utils.ByteArrayDataSource;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.zip.ZipInputStream;
+
+public class PdfHelper {
+
+    // Validate that data is a zipped pdf or throw an exception.
```
Comment (rkondratowicz) [inline src/test/java/uk/gov/hmcts/reform/sendletter/PdfHelper.java]: please use `/**`

File: src/integrationTest/java/uk/gov/hmcts/reform/sendletter/e2e/EndToEndTest.java (hunk 1 above)
Comment (banderous) [reply src/integrationTest/java/uk/gov/hmcts/reform/sendletter/e2e/EndToEndTest.java]: Can't throw checked exceptions from a method passed to map() unfortunately

Comment (rkondratowicz) [reply src/integrationTest/java/uk/gov/hmcts/reform/sendletter/e2e/EndToEndTest.java]: :love: java...

**Pull request merged**